heroic-component/src/main/java/com/spotify/heroic/metric/consts/ApiQueryConsts.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2015 Spotify AB.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.heroic.metric.consts;
+
+@SuppressWarnings({"LineLength"})
+public class ApiQueryConsts {
+    /**
+     * MutateRpcTimeoutMs
+     *
+     * The amount of milliseconds to wait before issuing a client side timeout
+     * for mutation remote procedure calls.
+     *
+     * Google default is 10 minutes (!!!)
+     *
+     * <p>Also defined as (source to be rediscovered...) :
+     * If timeouts are set, how many milliseconds should pass before a
+     * DEADLINE_EXCEEDED for a long mutation. Currently, this feature is experimental.
+     *
+     * @see <a href="https://github.com/googleapis/java-bigtable-hbase/blob/534288cfebf4732b7998c369a6e278581a64f758/bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java#L45">CallOptionsConfig.Builder#MutateRpcTimeoutMs</a>
+     */
+    public static final int DEFAULT_MUTATE_RPC_TIMEOUT_MS = 4_000;
+
+    /**
+     * ReadRowsRpcTimeoutMs
+     *
+     * The amount of milliseconds to wait before issuing a client side
+     * timeout for readRows streaming remote procedure calls.
+     *
+     * <p>AKA
+     * The default duration to wait before timing out read stream RPC
+     * (default value: 12 hour).
+     * @see <a href="https://github.com/googleapis/java-bigtable-hbase/blob/534288cfebf4732b7998c369a6e278581a64f758/bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java#L48">ReadRowsRpcTimeoutMs</a>
+     */
+    public static final int DEFAULT_READ_ROWS_RPC_TIMEOUT_MS = 4_000;
+
+    /**
+     * ShortRpcTimeoutMs
+     * The amount of milliseconds to wait before issuing a client side timeout for
+     * short remote procedure calls.
+     * TODO get from Adam proper description
```

Comment:
i would maybe avoid adding comments like this to open source

Reply:
👍🏻